src/PKSim.Core/Snapshots/Mappers/BaseClassificationMapper.cs

```diff
@@ -0,0 +1,69 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using OSPSuite.Core.Domain;
+
+namespace PKSim.Core.Snapshots.Mappers
+{
+   public abstract class BaseClassificationMapper<TSnapshot, TModelContext, TSnapshotContext, TClassifiable, TSnapshotClassifiable> : SnapshotMapperBase<Classification, TSnapshot, TModelContext, TSnapshotContext> 
```

Comment:
That's a lot of generics, but this allows for very little new code for other classification/classifiable mappers